src/main/java/com/example/store/dto/order/OrderDto.java

```diff
@@ -0,0 +1,17 @@
+package com.example.store.dto.order;
+
+import com.example.store.dto.orderitem.OrderItemDto;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+import lombok.Data;
+
+@Data
+public class OrderDto {
+    private Long id;
+    private Long userId;
+    private Set<OrderItemDto> orderItems;
+    private LocalDateTime orderDate;
+    private BigDecimal total;
+    private String status;
+}
```

Comment:
Please use consistent approach. Either all DTO's are records or classes. Otherwise you should check type every time you start to use new DTO class